src/pytest_celery/vendors/redis/container.py

```diff
@@ -44,7 +44,18 @@ def command(
         wait_for_client: bool = True,
         **kwargs: dict,
     ) -> list[str]:
-        return ["redis-server", *args]
+        return [
+            "redis-server",
+            "--save",
+            "",
+            "--appendonly",
+            "no",
+            "--maxmemory-policy",
+            "noeviction",
```

Comment:
```console
2024-09-13T11:51:55.0520459Z 1:C 13 Sep 2024 11:47:07.822 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
```